.github/workflows/auto-format.yml

```diff
@@ -0,0 +1,28 @@
+name: Auto-format
+
+on:
+  push:
+    branches: ['development']
+  pull_request:
+    branches: ['development']
```

Comment:
Shouldn't this be for all branches and only in PRs? I also wonder if this will get too annoying if you're working on a draft PR and start needing to pull commits, but I guess we'll find out 😂 
```suggestion
  pull_request:
    branches: *
```

Reply:
Ok. I really have no idea how this part of the yaml works. But, when I had this it worked on my fork with a branch that wasn't development. Does what you suggested mean a pull request into any branch and mine say a pull request into only development? Also,  does pull_request mean it runs anytime there is a change to a pull request.. I was thinking the "Push" was there to also run on new pushes to a pull request?